webroot/js/toolbar.js

```diff
@@ -97,6 +97,13 @@ if (elem) {
     doc.addEventListener('DOMContentLoaded', onReady, false);
     doc.addEventListener('DOMContentLoaded', proxyAjaxOpen, false);
     doc.addEventListener('DOMContentLoaded', proxyAjaxSend, false);
+
+    if (!window.__debugKitTurbolinksListenerApplied) {
+      doc.addEventListener('turbolinks:load', onReady, false);
+      doc.addEventListener('turbolinks:load', proxyAjaxOpen, false);
+      doc.addEventListener('turbolinks:load', proxyAjaxSend, false);
```

Comment:
These are the required listeners

Reply:
I guess they are requied by turbolinks. Could you please add a link in a comment on top of these, please?